website/versioned_docs/version-1.1/advanced/compose_api.md

```diff
@@ -111,3 +111,6 @@ def initialize_config_dir(config_dir: str, job_name: str = "app") -> None:
     """
 ```
 
+For an idea about how to modify Hydra's search path when using `compose` in
+unit tests, see the page on
+[overriding the `hydra.searchpath` config](search_path.md#overriding-hydrasearchpath-config).
```

Comment:
Thank you! Why specifically mention unit tests? This feature is more for a general purpose - making configs available in other dirs.

Reply:
Maybe the compose doc isn't the right place for the searchpath hint; searchpath modification is not just relevant for `compose`, but also for `@hydra.main` (which is the alternative to compose).

How about I move this unit test hint to the page on [unit testing](https://hydra.cc/docs/advanced/unit_testing/)?

Reply:
See the latest commit (https://github.com/facebookresearch/hydra/pull/2007/commits/130bde1bcf1595c608209de8fe1fede16a7591e9).